py/selenium/webdriver/firefox/webdriver.py

```diff
@@ -237,6 +237,8 @@ def install_addon(self, path, temporary=None):
         Returns identifier of installed addon. This identifier can later
         be used to uninstall addon.
 
+        :param path: Full path to the addon that will be installed.
+
```

Comment:
Can you update the `Usage` to be an absolute path as well? Something like `/path/to/firebug.xpi`